### ISSUE TITLE
Latest 15 Papers - May 02, 2025

### ISSUE THREAD

**Issue by KMnO4-zx** [KMnO4-zx/DailyArXiv#14]
**Please check the [Github](https://github.com/KMnO4-zx/DailyArXiv) page for a better reading experience and more papers.**

## Geoscience
| **Title** | **Date** | **KiMi** | **Comment** |
| --- | --- | --- | --- |
| **[GeoRSMLLM: A Multimodal Large Language Model for Vision-Language Tasks in Geoscience and Remote Sensing](http://arxiv.org/abs/2503.12490v1)** | 2025-03-16 | [papers-cool](https://papers.cool/arxiv/2503.12490v1) |  |
| **[Towards Foundation Models: Evaluation of Geoscience Artificial Intelligence with Uncertainty](http://arxiv.org/abs/2501.14809v1)** | 2025-01-15 | [papers-cool](https://papers.cool/arxiv/2501.14809v1) |  |
| **[When Geoscience Meets Foundation Models: Towards General Geoscience Artificial Intelligence System](http://arxiv.org/abs/2309.06799v5)** | 2024-11-12 | [papers-cool](https://papers.cool/arxiv/2309.06799v5) | <details><summary>accpe...</summary><p>accpeted by IEEE Geoscience and Remote Sensing Magazine</p></details> |
| **[Prototype-Based Methods in Explainable AI and Emerging Opportunities in the Geosciences](http://arxiv.org/abs/2410.19856v1)** | 2024-10-22 | [papers-cool](https://papers.cool/arxiv/2410.19856v1) | <details><summary>Accep...</summary><p>Accepted at AI for Science Workshop-Oral (Attention Track), Proceedings of 41st International Conference on Machine Learning (ICML) 2024</p></details> |
| **[Interpretable Geoscience Artificial Intelligence (XGeoS-AI): Application to Demystify Image Recognition](http://arxiv.org/abs/2311.04940v2)** | 2024-05-07 | [papers-cool](https://papers.cool/arxiv/2311.04940v2) | <details><summary>there...</summary><p>there are some erros in the results, and a newer revision is still preparing</p></details> |
| **[GeoGalactica: A Scientific Large Language Model in Geoscience](http://arxiv.org/abs/2401.00434v2)** | 2024-04-13 | [papers-cool](https://papers.cool/arxiv/2401.00434v2) |  |
| **[A novel approach for estimating functions in the multivariate setting based on an adaptive knot selection for B-splines with an application to a chemical system used in geoscience](http://arxiv.org/abs/2306.00686v3)** | 2024-01-25 | [papers-cool](https://papers.cool/arxiv/2306.00686v3) | 29 pages, 27 figures |
| **[When Geoscience Meets Generative AI and Large Language Models: Foundations, Trends, and Future Challenges](http://arxiv.org/abs/2402.03349v1)** | 2024-01-25 | [papers-cool](https://papers.cool/arxiv/2402.03349v1) |  |
| **[Differentiable modeling to unify machine learning and physical models and advance Geosciences](http://arxiv.org/abs/2301.04027v2)** | 2023-12-27 | [papers-cool](https://papers.cool/arxiv/2301.04027v2) |  |
| **[K2: A Foundation Language Model for Geoscience Knowledge Understanding and Utilization](http://arxiv.org/abs/2306.05064v2)** | 2023-09-13 | [papers-cool](https://papers.cool/arxiv/2306.05064v2) |  |
| **[AI Security for Geoscience and Remote Sensing: Challenges and Future Trends](http://arxiv.org/abs/2212.09360v2)** | 2023-06-22 | [papers-cool](https://papers.cool/arxiv/2212.09360v2) |  |
| **[Coarse-to-fine Task-driven Inpainting for Geoscience Images](http://arxiv.org/abs/2211.11059v3)** | 2022-12-06 | [papers-cool](https://papers.cool/arxiv/2211.11059v3) |  |
| **[Investigating the fidelity of explainable artificial intelligence methods for applications of convolutional neural networks in geoscience](http://arxiv.org/abs/2202.03407v2)** | 2022-09-05 | [papers-cool](https://papers.cool/arxiv/2202.03407v2) |  |
| **[Data Centred Intelligent Geosciences: Research Agenda and Opportunities, Position Paper](http://arxiv.org/abs/2209.02384v1)** | 2022-08-20 | [papers-cool](https://papers.cool/arxiv/2209.02384v1) |  |
| **[Beyond Visuals : Examining the Experiences of Geoscience Professionals With Vision Disabilities in Accessing Data Visualizations](http://arxiv.org/abs/2207.13220v1)** | 2022-07-27 | [papers-cool](https://papers.cool/arxiv/2207.13220v1) |  |

## Spatial
| **Title** | **Date** | **KiMi** | **Comment** |
| --- | --- | --- | --- |
| **[SDWPF: A Dataset for Spatial Dynamic Wind Power Forecasting Challenge at KDD Cup 2022](http://arxiv.org/abs/2208.04360v2)** | 2025-04-30 | [papers-cool](https://papers.cool/arxiv/2208.04360v2) | <details><summary>This ...</summary><p>This is a journal paper published in Scientific Data, titled SDWPF: A Dataset for Spatial Dynamic Wind Power Forecasting over a Large Turbine Array. The released sdwpf_full dataset provides more comprehensive information than the dataset made available for the KDD Cup 2022</p></details> |
| **[SpaRE: Enhancing Spatial Reasoning in Vision-Language Models with Synthetic Data](http://arxiv.org/abs/2504.20648v1)** | 2025-04-29 | [papers-cool](https://papers.cool/arxiv/2504.20648v1) |  |
| **[LiLIS: Enhancing Big Spatial Data Processing with Lightweight Distributed Learned Index](http://arxiv.org/abs/2504.18883v2)** | 2025-04-29 | [papers-cool](https://papers.cool/arxiv/2504.18883v2) |  |
| **[A spatial-correlated multitask linear mixed-effects model for imaging genetics](http://arxiv.org/abs/2407.04530v3)** | 2025-04-29 | [papers-cool](https://papers.cool/arxiv/2407.04530v3) | 32 pages, 5 figures |
| **[ISDrama: Immersive Spatial Drama Generation through Multimodal Prompting](http://arxiv.org/abs/2504.20630v1)** | 2025-04-29 | [papers-cool](https://papers.cool/arxiv/2504.20630v1) |  |
| **[MambaMoE: Mixture-of-Spectral-Spatial-Experts State Space Model for Hyperspectral Image Classification](http://arxiv.org/abs/2504.20509v1)** | 2025-04-29 | [papers-cool](https://papers.cool/arxiv/2504.20509v1) |  |
| **[SFIBA: Spatial-based Full-target Invisible Backdoor Attacks](http://arxiv.org/abs/2504.21052v1)** | 2025-04-29 | [papers-cool](https://papers.cool/arxiv/2504.21052v1) |  |
| **[STAA-SNN: Spatial-Temporal Attention Aggregator for Spiking Neural Networks](http://arxiv.org/abs/2503.02689v3)** | 2025-04-29 | [papers-cool](https://papers.cool/arxiv/2503.02689v3) | <details><summary>Accep...</summary><p>Accepted by CVPR 2025</p></details> |
| **[PRISM-DP: Spatial Pose-based Observations for Diffusion-Policies via Segmentation, Mesh Generation, and Pose Tracking](http://arxiv.org/abs/2504.20359v1)** | 2025-04-29 | [papers-cool](https://papers.cool/arxiv/2504.20359v1) |  |
| **[Dynamic Contextual Attention Network: Transforming Spatial Representations into Adaptive Insights for Endoscopic Polyp Diagnosis](http://arxiv.org/abs/2504.20306v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.20306v1) | <details><summary>Accep...</summary><p>Accepted at 47th Annual International Conference of the IEEE Engineering in Medicine and Biology Society (EMBC) 2025</p></details> |
| **[SpatialReasoner: Towards Explicit and Generalizable 3D Spatial Reasoning](http://arxiv.org/abs/2504.20024v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.20024v1) | <details><summary>Proje...</summary><p>Project page: https://spatial-reasoner.github.io</p></details> |
| **[Spatial von-Mises Fisher Regression for Directional Data](http://arxiv.org/abs/2207.08321v4)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2207.08321v4) |  |
| **[STCOcc: Sparse Spatial-Temporal Cascade Renovation for 3D Occupancy and Scene Flow Prediction](http://arxiv.org/abs/2504.19749v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.19749v1) |  |
| **[GPA-RAM: Grasp-Pretraining Augmented Robotic Attention Mamba for Spatial Task Learning](http://arxiv.org/abs/2504.19683v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.19683v1) |  |
| **[Generalizing Egocentric Temporal Neighborhoods to probe for spatial correlations in temporal networks and infer their topology](http://arxiv.org/abs/2501.16070v2)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2501.16070v2) |  |

## Time
| **Title** | **Date** | **KiMi** | **Comment** |
| --- | --- | --- | --- |
| **[Optimal Dynamic Control of Bounded Jacobian Discrete-Time Systems via Interval Observers](http://arxiv.org/abs/2504.21832v1)** | 2025-04-30 | [papers-cool](https://papers.cool/arxiv/2504.21832v1) | <details><summary>6 pag...</summary><p>6 pages, accepted in ECC</p></details> |
| **[Message Optimality and Message-Time Trade-offs for APSP and Beyond](http://arxiv.org/abs/2504.21781v1)** | 2025-04-30 | [papers-cool](https://papers.cool/arxiv/2504.21781v1) | <details><summary>Accep...</summary><p>Accepted to PODC 2025, abstract shortened to fit arXiv constraints</p></details> |
| **[An Adaptive Learning Approach to Multivariate Time Forecasting in Industrial Processes](http://arxiv.org/abs/2403.07554v2)** | 2025-04-30 | [papers-cool](https://papers.cool/arxiv/2403.07554v2) | 25 pages, 5 figures |
| **[Probabilistic Time Series Forecasting of Residential Loads -- A Copula Approach](http://arxiv.org/abs/2504.21661v1)** | 2025-04-30 | [papers-cool](https://papers.cool/arxiv/2504.21661v1) | <details><summary>Accep...</summary><p>Accepted for IEEE PowerTech</p></details> |
| **[Conditional independence testing with a single realization of a multivariate nonstationary nonlinear time series](http://arxiv.org/abs/2504.21647v1)** | 2025-04-30 | [papers-cool](https://papers.cool/arxiv/2504.21647v1) |  |
| **[Real Time Semantic Segmentation of High Resolution Automotive LiDAR Scans](http://arxiv.org/abs/2504.21602v1)** | 2025-04-30 | [papers-cool](https://papers.cool/arxiv/2504.21602v1) |  |
| **[Real-time Program Evaluation using Anytime-valid Rank Tests](http://arxiv.org/abs/2504.21595v1)** | 2025-04-30 | [papers-cool](https://papers.cool/arxiv/2504.21595v1) |  |
| **[Masked Diffusion Models are Secretly Time-Agnostic Masked Models and Exploit Inaccurate Categorical Sampling](http://arxiv.org/abs/2409.02908v6)** | 2025-04-30 | [papers-cool](https://papers.cool/arxiv/2409.02908v6) | <details><summary>Accep...</summary><p>Accepted at ICLR 2025</p></details> |
| **[Monotonicity and convergence of two-relaxation-times lattice Boltzmann schemes for a non-linear conservation law](http://arxiv.org/abs/2501.07934v2)** | 2025-04-30 | [papers-cool](https://papers.cool/arxiv/2501.07934v2) |  |
| **[Data Augmentation as Free Lunch: Exploring the Test-Time Augmentation for Sequential Recommendation](http://arxiv.org/abs/2504.04843v3)** | 2025-04-30 | [papers-cool](https://papers.cool/arxiv/2504.04843v3) | <details><summary>Accep...</summary><p>Accepted by SIGIR 2025 Full Paper</p></details> |
| **[Uniform-in-time weak error estimates of explicit full-discretization schemes for SPDEs with non-globally Lipschitz coefficients](http://arxiv.org/abs/2504.21364v1)** | 2025-04-30 | [papers-cool](https://papers.cool/arxiv/2504.21364v1) |  |
| **[R-ACP: Real-Time Adaptive Collaborative Perception Leveraging Robust Task-Oriented Communications](http://arxiv.org/abs/2410.04168v4)** | 2025-04-30 | [papers-cool](https://papers.cool/arxiv/2410.04168v4) | <details><summary>This ...</summary><p>This work has been accepted by IEEE JSAC. The code will be publicly available at: github.com/fangzr/R-ACP</p></details> |
| **[Filtering with Time-frequency Analysis: An Adaptive and Lightweight Model for Sequential Recommender Systems Based on Discrete Wavelet Transform](http://arxiv.org/abs/2503.23436v3)** | 2025-04-30 | [papers-cool](https://papers.cool/arxiv/2503.23436v3) | <details><summary>17pag...</summary><p>17pages, accepted by ICIC 2025 oral</p></details> |
| **[Hurdle Network Model With Latent Dynamic Shrinkage For Enhanced Edge Prediction in Zero-Inflated Directed Network Time Series](http://arxiv.org/abs/2504.21275v1)** | 2025-04-30 | [papers-cool](https://papers.cool/arxiv/2504.21275v1) |  |
| **[LSEAttention is All You Need for Time Series Forecasting](http://arxiv.org/abs/2410.23749v8)** | 2025-04-30 | [papers-cool](https://papers.cool/arxiv/2410.23749v8) | <details><summary>8 pag...</summary><p>8 pages with referencing, 1 figure, 5 tables</p></details> |

## Trajectory
| **Title** | **Date** | **KiMi** | **Comment** |
| --- | --- | --- | --- |
| **[Stable Trajectory Clustering: An Efficient Split and Merge Algorithm](http://arxiv.org/abs/2504.21808v1)** | 2025-04-30 | [papers-cool](https://papers.cool/arxiv/2504.21808v1) |  |
| **[Iterative Trajectory Exploration for Multimodal Agents](http://arxiv.org/abs/2504.21561v1)** | 2025-04-30 | [papers-cool](https://papers.cool/arxiv/2504.21561v1) | 16 pages, 8 figures |
| **[OS-Genesis: Automating GUI Agent Trajectory Construction via Reverse Task Synthesis](http://arxiv.org/abs/2412.19723v2)** | 2025-04-30 | [papers-cool](https://papers.cool/arxiv/2412.19723v2) | Work in progress |
| **[Estimation-Aware Trajectory Optimization with Set-Valued Measurement Uncertainties](http://arxiv.org/abs/2501.09192v2)** | 2025-04-30 | [papers-cool](https://papers.cool/arxiv/2501.09192v2) | 40 pages, 9 figures |
| **[Metaheuristic Optimization of Trajectory and Dynamic Time Splitting for UAV Communication Systems](http://arxiv.org/abs/2504.20425v1)** | 2025-04-29 | [papers-cool](https://papers.cool/arxiv/2504.20425v1) | <details><summary>5 pag...</summary><p>5 pages, 5 figures. Accepted by IEEE Communications Letters</p></details> |
| **[The Mean of Multi-Object Trajectories](http://arxiv.org/abs/2504.20391v1)** | 2025-04-29 | [papers-cool](https://papers.cool/arxiv/2504.20391v1) |  |
| **[Improving trajectory continuity in drone-based crowd monitoring using a set of minimal-cost techniques and deep discriminative correlation filters](http://arxiv.org/abs/2504.20234v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.20234v1) | <details><summary>Prepr...</summary><p>Preprint submitted to the Expert Systems with Applications journal</p></details> |
| **[Kinodynamic Trajectory Following with STELA: Simultaneous Trajectory Estimation & Local Adaptation](http://arxiv.org/abs/2504.20009v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.20009v1) | [Accepted] RSS 2025 |
| **[Towards Ball Spin and Trajectory Analysis in Table Tennis Broadcast Videos via Physically Grounded Synthetic-to-Real Transfer](http://arxiv.org/abs/2504.19863v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.19863v1) | <details><summary>To be...</summary><p>To be published in 2025 IEEE/CVF International Conference on Computer Vision and Pattern Recognition Workshops (CVPRW)</p></details> |
| **[UTTG_ A Universal Teleoperation Approach via Online Trajectory Generation](http://arxiv.org/abs/2504.19736v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.19736v1) |  |
| **[Geo-Llama: Leveraging LLMs for Human Mobility Trajectory Generation with Spatiotemporal Constraints](http://arxiv.org/abs/2408.13918v4)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2408.13918v4) |  |
| **[ARTEMIS: Autoregressive End-to-End Trajectory Planning with Mixture of Experts for Autonomous Driving](http://arxiv.org/abs/2504.19580v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.19580v1) |  |
| **[An End-to-End Framework for Optimizing Foot Trajectory and Force in Dry Adhesion Legged Wall-Climbing Robots](http://arxiv.org/abs/2504.19448v1)** | 2025-04-28 | [papers-cool](https://papers.cool/arxiv/2504.19448v1) |  |
| **[Trajectory Planning with Model Predictive Control for Obstacle Avoidance Considering Prediction Uncertainty](http://arxiv.org/abs/2504.19193v1)** | 2025-04-27 | [papers-cool](https://papers.cool/arxiv/2504.19193v1) | <details><summary>This ...</summary><p>This work has been accepted to IFAC for publication under a Creative Commons Licence CC-BY-NC-ND</p></details> |
| **[Efficient Control Allocation and 3D Trajectory Tracking of a Highly Manoeuvrable Under-actuated Bio-inspired AUV](http://arxiv.org/abs/2504.19049v1)** | 2025-04-26 | [papers-cool](https://papers.cool/arxiv/2504.19049v1) |  |
